internal/conf/computed.go

```diff
@@ -26,9 +27,20 @@ func init() {
 		log.Fatalf("The 'DEPLOY_TYPE' environment variable is invalid. Expected one of: %q, %q, %q, %q, %q, %q, %q. Got: %q", deploy.Kubernetes, deploy.DockerCompose, deploy.PureDocker, deploy.SingleDocker, deploy.Dev, deploy.Helm, deploy.App, deployType)
 	}
 
+	ContributeValidator(validateExternalURLConfig)
 	confdefaults.Default = defaultConfigForDeployment()
 }
 
+func validateExternalURLConfig(cfg conftypes.SiteConfigQuerier) (problems Problems) {
+	if val := cfg.SiteConfig().ExternalURL; val != "" {
+		var err error
+		if _, err = url.Parse(val); err != nil {
+			problems = append(problems, NewSiteProblem("Could not parse `externalURL`."))
+		}
```

Comment:
nit: why the bonus err declaration? And should we include the error message 

```suggestion
		if _, err := url.Parse(val); err != nil {
			problems = append(problems, NewSiteProblem(fmt.Sprintf("Could not parse `externalURL`: %s", err)))
		}
```